packages/sights/package.json

```diff
@@ -31,12 +31,15 @@
     "README.md"
   ],
   "scripts": {
-    "prepare": "yarn run lint && yarn run build",
+    "prepare": "npm run lint && npm run build",
```

Comment:
why switch to npm here ?

Reply:
to be compatible with other package managers, here I used pnpm, and for `run` is enough and bundled with node so it will run with anything